fix(telemetry): missing data

Krakaw opened the pull request:
Description
---
Fixes #335 

Motivation and Context
---
The telemetry data was missing the version, mode and app_id. This was because of an errant instance of the AppConfig being initialized then left unused except for the telemetry_manager.

How Has This Been Tested?
---
I `println`'d the telemetry data to check the correct values were being sent.

What process can a PR reviewer use to test or verify this change?
---
Run the app, opt into telemetry, view the telemetry data being sent matches what you see on screen.

<!-- Checklist -->
<!-- 1. Is the title of your PR in the form that would make nice release notes? The title, excluding the conventional commit
tag, will be included exactly as is in the CHANGELOG, so please think about it carefully. -->


Breaking Changes
---

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

<!-- Does this include a breaking change? If so, include this line as a footer -->
<!-- BREAKING CHANGE: Description what the user should do, e.g. delete a database, resync the chain -->

stringhandler commented:
Seems fine

stringhandler reviewed (approved):
I don't understand how it fixes this, but happy to merge anyway

Krakaw commented:
> I don't understand how it fixes this, but happy to merge anyway

The config initialization was being shadowed. So the first initialization was being cloned and moved into the tokio worker for the telemetry. Then the actual config that the main app was using and updating was being initialized and passed into the main app.

I just removed the useless first initialization and moved the main config init to above the telemetry init.
